Examples/LambdaFunctions/scripts/package.sh

```diff
@@ -17,11 +17,15 @@ set -eu
 
 executable=$1
 
-target=.build/lambda/$executable
+target=".build/lambda/$executable"
 rm -rf "$target"
 mkdir -p "$target"
 cp ".build/release/$executable" "$target/"
-cp -Pv /usr/lib/swift/linux/lib*so* "$target"
+#cp -Pv /usr/lib/swift/linux/lib*so* "$target"
+# add the target deps based on ldd
+ldd ".build/release/$executable" | grep swift | awk '{print $3}' | xargs cp -Pv -t "$target"
```

Comment:
👀 this is the most interesting part

Reply:
I'm not sure, but could we copy the libicu libs directly (and not just the symlinks), if we specify `xargs cp -Lv -t "$target"`

From the `man cp`
```
      -L    If the -R option is specified, all symbolic links are followed.
      -P    If the -R option is specified, no symbolic links are followed.  This is the default.
```

cc @drexin